### ISSUE TITLE
Multipart vpk files not handled in wildcard dirs

### ISSUE THREAD

**Issue by Fennecai** [lasa01/io_import_vmf#113]
**Describe the bug**
a VMF I have doesn't import because VPK files mess up....or something, idk. My game configuration is correct but the addon fails to import because ``file is not VPK (invalid magic)``

**To Reproduce**
1. Import a vmf after configuring Portal 2 and it's DLCs
2. fire dumbledore
3. there you go, invalid magic.

**Expected behavior**
I expected my map to import like it usually has without issues, rather than Dumbledore getting frustrated and getting himself fired whilst casting the overly-complicated file-import spells required to do so; thus resulting in invalid magic which caused the entire complex to simply vanish. We tried to hire Dr. strange in his place but he was a little busy patching up the multiverse after Cave Johnson messed it up for the 80th time this week...

**Related files**
I cant upload it to Github for fear somebody will discover it and use my work without permission (black mesa can eat my bankrupt-), but that said, it happens on any map import anyways.

**Logs**
```
Indexing game files...
Python: Traceback (most recent call last):
  File "C:\Users\Owner\AppData\Roaming\Blender Foundation\Blender\3.1\scripts\addons\io_import_vmf\__init__.py", line 728, in execute
    fs = self.get_filesystem(context)
  File "C:\Users\Owner\AppData\Roaming\Blender Foundation\Blender\3.1\scripts\addons\io_import_vmf\__init__.py", line 475, in get_filesystem
    return game_def.get_indexed_filesystem()
  File "C:\Users\Owner\AppData\Roaming\Blender Foundation\Blender\3.1\scripts\addons\io_import_vmf\__init__.py", line 270, in get_indexed_filesystem
    fs_dict[self.name] = VMFFileSystem(data_dirs, data_paks, index_files=True)
  File "C:\Users\Owner\AppData\Roaming\Blender Foundation\Blender\3.1\scripts\addons\io_import_vmf\deps\vmfpy\fs.py", line 82, in __init__
    self.index_all()
  File "C:\Users\Owner\AppData\Roaming\Blender Foundation\Blender\3.1\scripts\addons\io_import_vmf\deps\vmfpy\fs.py", line 150, in index_all
    self._do_index(self.iter_all())
  File "C:\Users\Owner\AppData\Roaming\Blender Foundation\Blender\3.1\scripts\addons\io_import_vmf\deps\vmfpy\fs.py", line 124, in _do_index
    for path, info in index_iter:
  File "C:\Users\Owner\AppData\Roaming\Blender Foundation\Blender\3.1\scripts\addons\io_import_vmf\deps\vmfpy\fs.py", line 121, in iter_all
    yield from self.iter_paks()
  File "C:\Users\Owner\AppData\Roaming\Blender Foundation\Blender\3.1\scripts\addons\io_import_vmf\deps\vmfpy\fs.py", line 117, in iter_paks
    yield from self.iter_pak(pak_file)
  File "C:\Users\Owner\AppData\Roaming\Blender Foundation\Blender\3.1\scripts\addons\io_import_vmf\deps\vmfpy\fs.py", line 110, in iter_pak
    pak = vpk.open(pak_file, path_enc="latin1")
  File "C:\Users\Owner\AppData\Roaming\Blender Foundation\Blender\3.1\scripts\addons\io_import_vmf\deps\vpk\__init__.py", line 16, in open
    return VPK(*args, **kwargs)
  File "C:\Users\Owner\AppData\Roaming\Blender Foundation\Blender\3.1\scripts\addons\io_import_vmf\deps\vpk\__init__.py", line 255, in __init__
    self.read_header()
  File "C:\Users\Owner\AppData\Roaming\Blender Foundation\Blender\3.1\scripts\addons\io_import_vmf\deps\vpk\__init__.py", line 357, in read_header
    raise ValueError("File is not VPK (invalid magic)")
ValueError: File is not VPK (invalid magic)
```

**Screenshot**

![image](https://user-images.githubusercontent.com/10889213/168409065-a8b803cf-d19d-4c29-a56d-d2c9ffc49445.png)

**Details (please complete the following information):**
 - OS: Aperture Science Disk Operating System v. 3.11 (well no actually windows 10)
 - Blender Version: 3.1.1 release candidate (technically a custom build called k-cycles)
 - Addon Version: latest
 - Did you download a release or build the addon yourself: release
 - Related Game: Portal 2

**Additional context**
Please bear with my jokes. Despite the comical tone of this report; the error is real and i cant seem to figure out what the problem is.

**Comment by lasa01** [comment]
Usually the error means that you have VPK files in the game configuration that end with _001 or some other number. These files are not really VPK files, they just store content for the "real' VPK files, which have the same name but end in _dir. So the solution would be to just remove any VPK files that end in numbers and

**Comment by lasa01** [comment]
Sorry, misclicked, didn't mean to close yet.

**Comment by Fennecai** [comment]
I am well aware of how vpk files work. Thats not the problem though because i don't have any of the "part" vpks in the game config. i have only specified the actual vpk indexes (the _dir ones)...

though that said; i have specified a couple wildcard paths that happen to contain those "part" files within. that might be the problem...still worth fixing i think. the wildcard shouldn't be searching for files it cant even use

**Comment by lasa01** [comment]
Yeah that's most likely the issue, there's no check for that currently.

**Comment by lasa01** [comment]
This is fixed in Plumber 1.0.0 beta versions, you can try the latest version [here](https://github.com/lasa01/io_import_vmf/releases/tag/v1.0.0-beta3).
